src/components/Modal/Modal.js

```diff
@@ -77,25 +78,34 @@ class Modal extends React.Component {
     event.stopPropagation();
   };
 
+  renderTitle = () => {
+    const { title, onClose, Title } = this.props;
+    if (!title) return null;
+    return (
+      <Title>
+        {title}
+        <Anchor icon="delete2" onClick={onClose} />
```

Comment:
Adding a truthiness check for `onClose` to conditionally render the anchor would let us not render a close button that doesn't do anything.

Reply:
Also, to adhere to the 'overrideable props' rule, you'll want to make `CloseButton` a prop for this component and use it here, and provide the SCL anchor as a default prop.